src/common/string/slugify.ts

```diff
@@ -1,19 +1,25 @@
 // https://gist.github.com/hagemann/382adfc57adbd5af078dc93feef01fe1
 export const slugify = (value: string, delimiter = "_") => {
   const a =
-    "àáäâãåăæąçćčđďèéěėëêęğǵḧìíïîįłḿǹńňñòóöôœøṕŕřßşśšșťțùúüûǘůűūųẃẍÿýźžż·/_,:;";
-  const b = `aaaaaaaaacccddeeeeeeegghiiiiilmnnnnooooooprrsssssttuuuuuuuuuwxyyzzz${delimiter}${delimiter}${delimiter}${delimiter}${delimiter}${delimiter}`;
+    "àáâäæãåāăąçćčđďèéêëēėęěğǵḧîïíīįìıİłḿñńǹňôöòóœøōõőṕŕřßśšşșťțûüùúūǘůűųẃẍÿýžźż·";
+  const b = `aaaaaaaaaacccddeeeeeeeegghiiiiiiiilmnnnnoooooooooprrsssssttuuuuuuuuuwxyyzzz${delimiter}`;
   const p = new RegExp(a.split("").join("|"), "g");
 
-  return value
+  const slugified = value
     .toString()
     .toLowerCase()
-    .replace(/\s+/g, delimiter) // Replace spaces with delimiter
     .replace(p, (c) => b.charAt(a.indexOf(c))) // Replace special characters
-    .replace(/&/g, `${delimiter}and${delimiter}`) // Replace & with 'and'
-    .replace(/[^\w-]+/g, "") // Remove all non-word characters
-    .replace(/-/g, delimiter) // Replace - with delimiter
+    .replace(/(?<=\d),(?=\d)/g, "") // Remove Commas between numbers
+    .replace(/[^a-z0-9]+/g, delimiter) // Replace all non-word characters
     .replace(new RegExp(`(${delimiter})\\1+`, "g"), "$1") // Replace multiple delimiters with single delimiter
     .replace(new RegExp(`^${delimiter}+`), "") // Trim delimiter from start of text
     .replace(new RegExp(`${delimiter}+$`), ""); // Trim delimiter from end of text
+
+  if (value == "") {
+    return "";
```

Comment:
We can do this before calculating the slug

Reply:
That is correct, but it would be returning from 2 different places in the code, which I consider to be ugly. The time and space complexity are the same in that case (because it is the empty string) and this is easier to read in my opinion.

Reply:
Nvm. I just realized that the linter wasn't happy with the "else". I changed it so that it checked "value" before calculating the slug.